tests/ssl-opt.sh

```diff
@@ -2070,6 +2070,54 @@ run_test    "SNI: CA override with CRL" \
             -S "! The certificate is not correctly signed by the trusted CA" \
             -s "The certificate has been revoked (is on a CRL)"
 
+# Tests for SNI and DTLS
+
+run_test    "SNI/DTLS: matching cert 1" \
```

Comment:
For consistency, the description should be `SNI: DTLS, matching cert 1`. Not a blocker.

Reply:
I have renamed the tests for consistency.